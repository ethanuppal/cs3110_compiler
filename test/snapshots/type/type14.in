@ffi puts(Char*)

func main() {
    let val = "hi"
}
