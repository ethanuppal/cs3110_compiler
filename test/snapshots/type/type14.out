@ffi puts(Char*) -> Unit
func main() -> Unit {
    let val: Char* = "hi"
    return
}
